go.mod

```diff
@@ -1,38 +1,79 @@
 module github.com/invidian/terraform-provider-gpg
 
-go 1.15
+go 1.17
+
+require (
+	github.com/hashicorp/terraform-plugin-sdk v1.15.0
+	golang.org/x/crypto v0.0.0-20200728195943-123391ffb6de
```

Comment:
The latest version of this library is marked as deprecated, which breaks the linter. So I decided to keep the same version as before, and leave it for #9 to sort out crypto library upgrades.

Reply:
Ah, either this or I was thinking about suppressing the linter by a configuration line. Example: https://github.com/flexkube/libflexkube/blob/7717fd43362d6ca923f9d974485cf4af2b59893d/.golangci.yml#L14-L18